### DISCUSSION TITLE
Prevent background from being scrolled when a modal is open.

### DISCUSSION THREAD

**Discussion by seedprod** [livewire/flux#291] [Feature Requests] [answered]
Right now when a modal is open the background is scrollable. It would be nice to prevent this by default.

**Comment by joffreypersia** [comment]
+1

**Comment by helderneves91** [comment]
In most of my applications it's actually good to scroll, in others I would like to disable that...

**Comment by seedprod** [comment]
Workaround:

`        /* prevent scroll when dialog is open */
        body:has(dialog[open]) {
            @apply overflow-hidden; 
        }
`

**Comment by joshhanley** [comment, marked as answer]
Going to close this discussion as this was implemented in Flux v1.0.29.
